config/webpack.config.dev.js

```diff
@@ -1,4 +1,3 @@
-'use strict';
 
```

Comment:
Empty line.
We should add "maxBOF": 0 to .eslintrc on no-multiple-empty-lines.

Reply:
Resolved